appveyor_build.sh

```diff
@@ -41,33 +41,34 @@ function set_configuration {
     echo "Edit $FILE to set PREFIX=$2"
     sed -e "/PREFIX=/s|=.*|=$2|" \
         -e "/^ *CFLAGS *=/s/\r\?$/ $3\0/" \
-         config/Makefile.$1 > config/Makefile
+         "config/Makefile.$1" > config/Makefile
 #    run "Content of $FILE" cat config/Makefile
 }
 
-APPVEYOR_BUILD_FOLDER=$(echo $APPVEYOR_BUILD_FOLDER| cygpath -f -)
+APPVEYOR_BUILD_FOLDER=$(echo "$APPVEYOR_BUILD_FOLDER" | cygpath -f -)
 # These directory names are specified here, because getting UTF-8 correctly
 # through appveyor.yml -> Command Script -> Bash is quite painful...
-OCAMLROOT=$(echo $PROGRAMFILES/Бактріан🐫| cygpath -f - -m)
+OCAMLROOT=$(echo "$PROGRAMFILES/Бактріан🐫" | cygpath -f - -m)
 
 # This must be kept in sync with appveyor_build.cmd
 BUILD_PREFIX=🐫реализация
 
-export PATH=$(echo $OCAMLROOT| cygpath -f -)/bin/flexdll:$PATH
+PATH=$(echo "$OCAMLROOT" | cygpath -f -)/bin/flexdll:$PATH
+export PATH
 
 case "$1" in
   install)
     mkdir -p "$OCAMLROOT/bin/flexdll"
-    cd $APPVEYOR_BUILD_FOLDER/../flexdll
+    cd "$APPVEYOR_BUILD_FOLDER/../flexdll" || exit 1
```

Comment:
This script is run with `bash -e`

Reply:
Should be moved to this script then. As for the travis one. Updated.